workflow/resources/datavzrd/go-enrichment-template.yaml

```
@@ -133,3 +133,12 @@ views:
               }  
             spec-path: ?input.vega_waterfall
             vega-controls: true
+  go_terms_scatter:
+    dataset: go_enrichment
+    desc: ?f"The y-axis represents the False Discovery Rate (FDR), while the x-axis shows the effect of the GO term for the {wildcards.model} model as determined by goatools.
+    The effect is determined as the sum of all the absolute ??XYZ?? values across all genes in a GO term.
```

This assumes that we stick with the sum of fold changes:
```suggestion
    The effect is determined as the sum of all the absolute fold changes (on a log2 scale) across all genes in a GO term.
```
Alternatively, we could go for an average of the fold changes. I'll open a pull request for this effect calculcation, so we can discuss and decide there.

---

Then this PR should be fine :)

---

This should be addressed by PR #141, and the respective adjustment of this description that I did in this commit:
https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/pull/136/commits/fda25ee840951bd934cc9aa4458315655bf68c31